lib/api/v3/queries/queries_api.rb

```diff
@@ -88,10 +90,16 @@ def allowed_to?(action)
             end
           end
 
+          post do
+            create_query request_body, current_user
+          end
+
           params do
             requires :id, desc: 'Query id'
           end
           route_param :id do
+            mount API::V3::Queries::UpdateFormAPI
```

Comment:
this has to be placed after the `before` block in order for the query to be found